transpiler/src/zero_generator.rs

```diff
@@ -136,33 +142,55 @@ impl Transpiler {
         self.add_instruction(GeneralInstruction::Pseudo(jump_inst));
     }
 
+    fn add_jump_if(&mut self, cond: LocalOrImmediate, addr: ImmediateOrMacro) {
```

Comment:
Hmm I was imagining that mostly we'd just construct instructions and then call the more generic `add_instruction`; but this way works too! There's pros and cons

Reply:
sounds good; we can always shift back if need be